frontend/src/lib/Quiver.ts

```diff
@@ -71,21 +71,25 @@ type Data = DataType[][]
 /** DataFrame index and data types. */
 interface Types {
   /** Types for each index column. */
-  index: IndexType[]
+  index: Type[]
 
   /** Types for each data column. */
   // NOTE: `DataTypeName` should be used here, but as it's hard (maybe impossible)
   // to define such recursive types in TS, `string` will suffice for now.
-  data: string[]
+  data: Type[]
 }
 
-/** Type information for a single index column. */
-interface IndexType {
+/** Type information for a single index and data columns. */
```

Comment:
This docstring is slightly confusing, can you change it to whichever of these is more correct:

"Type information for index and data columns" or
"Type information for single-index columns, and data columns"